app/Comments.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Comments extends Model
+{
+    protected $table = 'comments';
+    const LIMIT_COMMENT = 5;
```

Comment:
Can all const group together in 1 place instead of each file contain 1

Reply:
I fixed